common4j/src/main/com/microsoft/identity/common/java/AuthenticationConstants.java

```diff
@@ -514,6 +514,11 @@ public static final class Broker {
          */
         public static final long BROKER_TASK_DEFAULT_TIMEOUT_MILLISECONDS = TimeUnit.SECONDS.toMillis(30);
 
+        /**
+         * Timeout for DCF token request
+         */
+        public static final long DCF_TOKEN_REQUEST_TIMEOUT_MILLISECONDS = TimeUnit.MINUTES.toMillis(14);
```

Comment:
Why is the timeout set to 14mins?

Reply:
DCF token expiry is 15 minutes , so i set the timeout for 14 to take 1 minute as buffer ? so that we don't end up sending a code to the user that is expired or just about to expire...

Reply:
added comment regarding the above